wpeverest/everest-forms
# Enhancement - Logger

### All Submissions:



### Changes proposed in this Pull Request:

Previously, there was no logging mechanism while saving form and submitting form. This PR enhance this feature.

### How to test the changes in this Pull Request:

1. Go to the Everest forms settings > general settings >  Enable storing the logs.
2. Create form and save this.
3. Create some entries.
4. Go to the Everest forms tools > logs
5. Verify generated logs.

### Types of changes:


### Other information:


### Changelog entry

> Enhancement - Logger
